kaitai_utils.py

```diff
@@ -44,6 +46,28 @@ def _get_metadata():
         return yaml.safe_load(file)['meta']
 
 
+def search_process_key(object):
```

Comment:
PyCharm says "Shadows built-in name 'object'", so it might be good to use a different parameter name here.

Reply:
changed!